src/ints/mouse.cpp

```diff
@@ -555,6 +556,7 @@ void Mouse_ButtonReleased(Bit8u button) {
 	case 0:
 		if (!(mouse.buttons&1)) return;
 		mouse.buttons&=~1;
+		LOG_MSG("Mouse_AddEvent(MOUSE_LEFT_RELEASED)");
```

Comment:
Same with this log.

Reply:
Thanks for catching this @bmunger !
These logs were added to help @Burrito78 and I debug the the lost clicks problem. It helped indicate when clicks were registered from the OS into SDL. 
The plan was to knock these out in the release version; so will do that shortly 👍